packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -607,6 +613,11 @@ export class GarbageCollector implements IGarbageCollector {
 			}
 			this.sweepTimeoutMs =
 				testOverrideSweepTimeoutMs ?? computeSweepTimeout(this.sessionExpiryTimeoutMs);
+			if (this.gcOptions[gcTombstoneGenerationOptionName] !== undefined) {
```

Comment:
I wonder if we make this.persistedGCFeatureMatrix always defined, that way this if statement can be simplified to a ??.

Reply:
It breaks snapshot tests.  I have a follow-up item opened.